targets/common.nix

```diff
@@ -0,0 +1,9 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Common modules for all targets
+{
+  imports = [
+    ../modules/graphics/weston.nix
```

Comment:
I would propose to keep common target headless and have graphics enabler separately.

Reply:
Very good point!

Reply:
I will do this

Reply:
I was about to do the change when the first headless target comes in, but we can of course do that beforehand